iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/GcpIpRestrictedAdminAccessCheckPart.java

```diff
@@ -40,19 +40,16 @@ public class GcpIpRestrictedAdminAccessCheckPart extends AbstractNewResourceChec
 
   private static final Set<String> SENSITIVE_PREFIXES = Set.of(ALL_IPV4, ALL_IPV6, "0::0/0", "::0/0");
 
-  private static final Predicate<ExpressionTree> RANGE_CONTAINS_SENSITIVE_PORTS =
-    range -> TextUtils.matchesValue(range, IpRestrictedAdminAccessCheck::rangeContainsSshOrRdpPort).isTrue();
-
-  private static final Predicate<ExpressionTree> SENSITIVE_IP_RANGE =
-    range -> TextUtils.matchesValue(range, SENSITIVE_PREFIXES::contains).isTrue();
+  private static final Predicate<ExpressionTree> RANGE_CONTAINS_SENSITIVE_PORTS = range -> TextUtils.matchesValue(range, IpRestrictedAdminAccessCheck::rangeContainsSshOrRdpPort)
```

Comment:
This line will be too long?
```suggestion
  private static final Predicate<ExpressionTree> RANGE_CONTAINS_SENSITIVE_PORTS = range -> TextUtils
      .matchesValue(range, IpRestrictedAdminAccessCheck::rangeContainsSshOrRdpPort)
```

Reply:
I think it's specific for the Eclipse formatter - it breaks lines sometimes in a strange way.
IntelliJ Idea by default (or there is a setting somewhere for it) respects new lines introduced by the author